packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -287,7 +293,7 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
                 onDropPositionChange={onDropPositionChange}
               />
             )}
-
+            <EventListener type="keydown" listener={handleKeyDown} target={document} />
```

Comment:
```suggestion
            <EventListener type="keydown" listener={onKeyDown} target={document} />
```
Do we really need the `handleKeyDown`?